etc/zinit/networkd.yaml

```diff
@@ -1,4 +1,5 @@
-exec: networkd --broker unix:///var/run/redis.sock --root /var/cache/modules/networkd
-test: zbusdebug --module network
+exec: netlightd --broker unix:///var/run/redis.sock --root /var/cache/modules/networkd
+
+# test: zbusdebug --module network
```

Comment:
We still have to have something here to let other modules know that this networking is ready. I am sure netlightd also has a "zbus" service which means we can use the same test command but with different module name (if the module name was changed even)